protocol/extend.go

```diff
@@ -1,13 +1,39 @@
 package protocol
 
-import "github.com/jackc/pgx/pgproto3"
+import (
+	"fmt"
+	"github.com/jackc/pgx/pgio"
+	"github.com/jackc/pgx/pgproto3"
+	nodes "github.com/lfittl/pg_query_go/nodes"
+)
 
 // ParseComplete is sent when backend parsed a prepared statement successfully
 var ParseComplete = []byte{'1', 0, 0, 0, 4}
 
 // BindComplete is sent when backend prepared a portal and finished planning the query
 var BindComplete = []byte{'2', 0, 0, 0, 4}
 
+// ParameterDescription is sent when backend received Describe message from frontend
+// with ObjectType = 'S' - requesting to describe prepared statement with a provided name
+func ParameterDescription(ps *nodes.PrepareStmt) (Message, error) {
+	res := []byte{'t'}
```

Comment:
I think that it's redundant because in the protocol, `t` is the indicator of `ParameterDescription` message type, and this is the method that builds this kind of message. we won't need to parse messages of this type, because it's a backend message, hence `'t'` won't be used with the same meaning outside of this context